src/test/should-compile/index-bounds.fuse

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
By default, the small example should go in TypeCheckerSpec.scala which allows us to track errors thrown and values expected in a more granular way. `should-compile` and `should-fail` are mostly for larger programs or benchmarks.